README.md

```diff
@@ -46,6 +46,30 @@ class Foo {
 }
 ```
 
+If you specify option `"scriptPath": true`:
+```js
+"plugins": [
+  ["annotate-console-log", {
+    "scriptPath": true
+  }]
+]
+```
+it'll add a short script path (and line number)
+```js
+console.log('apple')
+// =>
+console.log('[file.js:3]', 'foo()', 'apple')
+```
+Use `"fullScriptPath": true` to add more detailed script path info
```

Comment:
I like this idea. To clarify, `'fileName'` would be just `foo.js`, and `'relative'` would be relative to the project root, right? Should there also be an option `'fullpath'` for actual full path? You mentioned above "iterm2-open-text-file-paths", I'm not familiar with it but would it require full path?